src/react/databrowser/objects/ObjectHead.jsx

```diff
@@ -1,14 +1,17 @@
-// @flow
+// @noflow
```

Comment:
I can't flow type here because "Expected coverage to be at least 30%, but is: 23.33%(flowtype-errors/enforce-min-coverage)"
this gives me eslint error.
However, I update flow type usage in BucketHead.jsx